routes/web.php

```diff
@@ -198,6 +198,17 @@
     Route::permanentRedirect('demo', '/');
 });
 
+// routes that don't need authentication
+Route::group([], function () {
```

Comment:
Don't really need the groups here, you can just set it on one line (or one group).

Reply:
I've just simplified this config.